Material.Styles/Material.Styles.csproj

```diff
@@ -3,13 +3,14 @@
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-        <LangVersion>8</LangVersion>
+        <LangVersion>9</LangVersion>
```

Comment:
Interesting...
It will works properly on .NET Standard 2.0??

Reply:
Yes, it works. 
LangVersion affects only requirements for the dotnet version on the machine on which you build the project. Those are quite normal that in the future we will build projects for C# 25.0 for netstandard2.0 through .Net 15 for example